lib/util.js

```diff
@@ -129,6 +129,12 @@ function styleText(format, text, { validateStream = true, stream = process.stdou
     }
   }
 
+  const skipColorize = validateStream &&
+    (
+      !stream ||
+      !lazyUtilColors().shouldColorize(stream)
+    );
```

Comment:
We can move into https://github.com/nodejs/node/blob/08eeddfa8391fb8c085eb2d23ca155a9ae492d58/lib/util.js#L122-L130, and remove `!stream`.